modules/nf-core/universc/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Dockerfile to create container with Cell Ranger v3.0.2 and Universc v1.2.3
+# Push to nfcore/universc:<VER>
+
+FROM tomkellygenetics/universc:1.2.3
```

Comment:
Can we not just use `tomkellygenetics/universc:1.2.3`? I think the extra step of pushing it to nf-core is just more hassle, it should either live in nf-core or at `tomkellygenetics`.

Reply:
Sorry for the misunderstanding. I've removed the Dockerfile from the module directory. The external container `tomkellygenetics/universc:1.2.4` is already used. (note a newer version supports testing on small files with 10x parameters)